src/components/ConversationSettings/Matterbridge/MatterbridgeSettings.vue

```diff
@@ -36,22 +36,19 @@
 				<div v-show="!enabled"
 					class="add-part-wrapper">
 					<Plus class="icon" :size="20" />
-					<NcMultiselect ref="partNcMultiselect"
-						v-model="selectedType"
+					<NcSelect v-model="selectedType"
 						label="displayName"
-						track-by="type"
 						:placeholder="newPartPlaceholder"
 						:options="formatedTypes"
-						:user-select="true"
 						:internal-search="true"
```

Comment:
```suggestion
```

`searchable` now, which is true by default

Reply:
Apart from that, everything is fine, component is working, though it's intended to clear itself after picking